lib/view/Screens/AddressScreen/components/Addresslist.dart

```diff
@@ -24,6 +24,7 @@ class AddressList extends StatelessWidget {
                     padding: const EdgeInsets.only(bottom: 16.0),
                     child: AddressBox(
                         isSelected: index == controller.selectedAddressIndex,
```

Comment:
remove the isSelected argument from AddressBox

Reply:
Requested changes has been made !!